src/rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -177,17 +179,36 @@ function  azure_rdma_preparation()
 azure_rdma_preparation
 {{/ azRDMA }}
 {{/ reqAzRDMA }}
-# Write env to system-wide environment
-env | grep -E "^PAI|PATH|PREFIX|JAVA|HADOOP|NVIDIA|CUDA" > /etc/environment
+
+function write_user_command()
+{
+  PAI_USER_COMMAND_DIR="/pai/taskData"
+  PAI_USER_COMMAND_PATH="${PAI_USER_COMMAND_DIR}/command.sh"
+  mkdir -p "${PAI_USER_COMMAND_DIR}"
+
+  echo "#!/bin/bash" > "${PAI_USER_COMMAND_PATH}"
+
+  # By default, exit on the first error
+  echo 'set -e' >> "${PAI_USER_COMMAND_PATH}"
+  # Enable logging for the user command
+  echo 'set -x' >> "${PAI_USER_COMMAND_PATH}"
+
+  cat <<EOF >> "${PAI_USER_COMMAND_PATH}"
+{{{ taskData.command }}}
+EOF
```

Comment:
I've realized it should not be done like this - if the user command contains heredocs it can fail. So, I'll revert to the previous escaped variant.